recipes/google-cloud-cpp-core/meta.yaml

```
@@ -0,0 +1,387 @@
+  - name: libgoogle-cloud-policytroubleshooter-devel
+    {{ child_devel_subpackage("policytroubleshooter", "pubsub")|indent(4) }}
+
+  - name: libgoogle-cloud-storage
+    # This cannot use the standard subpackages because it has different direct
+    # dependencies: libcrc32c and libcurl being the most prominent ones.
+    script: install-libgoogle-cloud.sh   # [not win]
+    script: install-libgoogle-cloud.bat  # [win]
+    build:
+      run_exports:
+        # patch versions guaranteed to be ABI-compatible, see
+        # https://github.com/googleapis/google-cloud-cpp/blob/main/doc/adr/2023-05-03-patch-releases-are-ABI-compatible.md
+        - {{ pin_subpackage("libgoogle-cloud-storage", max_pin="x.x") }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+      host:
+        - {{ pin_subpackage("libgoogle-cloud", exact=True) }}
+        - libcrc32c
+        # This can be removed on the first release (likely v2.18.0) of
+        # google-cloud-cpp containing:
+        #   https://github.com/googleapis/google-cloud-cpp/pull/12890
+        - zlib
+      run:
+        - {{ pin_subpackage("libgoogle-cloud", exact=True) }}
+        - libabseil
+        - libcrc32c
+        - libcurl
+        - openssl
+        - libcxx  # [osx]
+    test:
+      {{ test_subpackage("storage")|indent(6) }}
+
+  - name: libgoogle-cloud-storage-devel
+    # This cannot use the standard subpackages because it has different direct
+    # dependencies: libcrc32c and libcurl being the most prominent ones.
+    script: install-libgoogle-cloud.sh   # [not win]
+    script: install-libgoogle-cloud.bat  # [win]
+    build:
+      run_exports:
+        # patch versions guaranteed to be ABI-compatible, see
+        # https://github.com/googleapis/google-cloud-cpp/blob/main/doc/adr/2023-05-03-patch-releases-are-ABI-compatible.md
+        - {{ pin_subpackage("libgoogle-cloud-storage", max_pin="x.x") }}
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - cmake
+        - ninja
+      host:
+        - {{ pin_subpackage("libgoogle-cloud-devel", exact=True) }}
+        - libabseil
+        - libcrc32c
+        - libcurl
+        - openssl
+        - nlohmann_json
+      run:
+        - {{ pin_subpackage("libgoogle-cloud-devel", exact=True) }}
+        - {{ pin_subpackage("libgoogle-cloud-storage", exact=True) }}
+        - libabseil
+        - libcrc32c
+        - libcurl
+        - openssl
+        - nlohmann_json
+        - libcxx  # [osx]
+    test:
+      {{ test_devel_subpackage("storage")|indent(6) }}
+
+about:
+  home: https://github.com/googleapis/google-cloud-cpp
+  license: Apache-2.0
+  license_family: Apache
+  license_file: LICENSE
+  summary: Google Cloud Client Library for C++
+
+extra:
+  feedstock-name: google-cloud-cpp-core
+  recipe-maintainers:
+    - coryan
+    - davidbrochart
+    - dbolduc
+    - h-vetinari
+    - xhochy
```

I think rather than have everyone involved reconfirm that they're up for maintainership, the best way would be to piggyback off of the existing feedstock
```suggestion
    - coryan
    - conda-forge/google-cloud-cpp
```

---

Done